tv_apps/tv-deck/js/contextmenu.js

```diff
@@ -0,0 +1,79 @@
+/* global PinCard, MozActivity */
+
+'use strict';
+
+(function(exports) {
+  function Contextmenu(buttonConfigs, app) {
```

Comment:
I am not sure which one we should use: Contextmenu or ContextMenu. I think they are two words. So, I prefer ContextMenu.